src/Component/Metadata/Delete.php

```diff
@@ -35,6 +35,9 @@ public function __construct(
         ?bool $write = null,
         ?string $formType = null,
         ?array $formOptions = null,
+        ?string $stateMachineComponent = null,
+        ?string $stateMachineTransition = null,
+        ?string $stateMachineGraph = null,
```

Comment:
I'm wondering if we support state machine transition on delete. Can you check?

Reply:
I see no state machine transition usage on deleteAction, but I only see state machine usage on createAction but that's weird.